release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1994,6 +1989,9 @@ With this release, when the `UPDATE_URL_OVERRIDE` environment variable is set, o
 
 * Previously, the `oc adm node-image create --pxe generated` command did not create only the Preboot Execution Environment (PXE) artifacts. Instead, the command created the PXE artifacts with other artifacts from a `node-joiner` pod and stored them all in the wrong subdirectory. Additionally, the PXE artifacts were incorrectly prefixed with `agent` instead of `node`. With this release, generated PXE artifacts are stored in the correct directory and receive the correct prefix. (link:https://issues.redhat.com/browse/OCPBUGS-46449[*OCPBUGS-46449*])
 
+* Previously, requests to the `deploymentconfig/scale` subresource would fail when there was an admission webhook matching the request. With this release, the issue is resolved and requests to the deploymentconfig/scale` subresource will succeed. (link:https://issues.redhat.com/browse/OCPBUGS-41136[*OCPBUGS-41136*]) 
```

Comment:
```suggestion
* Previously, requests to the `deploymentconfig/scale` subresource would fail when there was an admission webhook matching the request. With this release, the issue is resolved and requests to the `deploymentconfig/scale` subresource will succeed. (link:https://issues.redhat.com/browse/OCPBUGS-41136[*OCPBUGS-41136*]) 
```